package.json

```diff
@@ -17,6 +17,7 @@
     "@google-cloud/recaptcha-enterprise": "^5.10.0",
     "@marsidev/react-turnstile": "^1.0.2",
     "@martianwallet/aptos-wallet-adapter": "^0.0.4",
+    "@mizuwallet-sdk/core": "^1.4.0",
```

Comment:
These packages were added, but aren't needed in the application, can you elaborate on the purpose?